src/main/java/uk/gov/hmcts/darts/util/AzureCopyUtil.java

```diff
@@ -23,15 +23,18 @@ public void copy(String source, String destination) {
 
             var startTime = Instant.now();
             log.debug("copy of blob started at {}", startTime);
-
+            builder.redirectErrorStream(true);
             Process p = builder.start();
             int exitValue = p.waitFor();
             var endTime = Instant.now();
             log.debug("copy of blob completed at {}. Total duration in seconds: {}", endTime, Duration.between(startTime, endTime).getSeconds());
 
             if (exitValue != 0) {
+                String result = new String(p.getInputStream().readAllBytes());
+                log.error("Failed to execute azcopy from source: '{}' to destination '{}'- error exit value. Result: {}", source, destination, result);
+
                 //limiting info included in the exception to avoid potential leak of the source and destination sas tokens
-                throw new DartsException("Failed to execute azcopy - error exit value");
+//                throw new DartsException("Failed to execute azcopy - error exit value");
```

Comment:
Not that you'll forget to uncomment this when the issue is fixed :D but suggest adding a TODO comment with the ticket ref to ensure it's never missed.

Reply:
will sonar complain about TODO comments?